CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.6.1
+
+**Release date:** 2021-01-14
+
+This prerelease comes with support for triggering 
+container image updates to Git using webhook receiver.
```

Comment:
Would maybe mention the regression bug here too, as I did this for all controllers that already had seen a release.